### ISSUE TITLE
Configure bors-ng for all projects

### ISSUE THREAD

**Issue by vonglasow** [hoaproject/Central#81]
First allow the application in the settings

Add bors.toml with right configuration on all libraries listed below:

- [ ] [Acl](https://github.com/hoaproject/Acl)
- [ ] [Bench](https://github.com/hoaproject/Bench)
- [ ] [Blog](https://github.com/hoaproject/Blog)
- [ ] [Cache](https://github.com/hoaproject/Cache)
- [ ] [Central](https://github.com/hoaproject/Central)
- [ ] [Cli](https://github.com/hoaproject/Cli)
- [ ] [Compiler](https://github.com/hoaproject/Compiler)
- [ ] [Consistency](https://github.com/hoaproject/Consistency)
- [ ] [Console](https://github.com/hoaproject/Console)
- [ ] [Contributions-Atom-Pp](https://github.com/hoaproject/Contributions-Atom-Pp)
- [ ] [Contributions-Atoum-PraspelExtension](https://github.com/hoaproject/Contributions-Atoum-PraspelExtension)
- [ ] [Contributions-Emacs-Pp](https://github.com/hoaproject/Contributions-Emacs-Pp)
- [ ] [Contributions-Provisioning](https://github.com/hoaproject/Contributions-Provisioning)
- [ ] [Contributions-Symfony-BenchBundle](https://github.com/hoaproject/Contributions-Symfony-BenchBundle)
- [ ] [Contributions-Symfony-ConsoleBridge](https://github.com/hoaproject/Contributions-Symfony-ConsoleBridge)
- [ ] [Contributions-Symfony-ConsoleBundle](https://github.com/hoaproject/Contributions-Symfony-ConsoleBundle)
- [ ] [Contributions-Symfony-RulerBundle](https://github.com/hoaproject/Contributions-Symfony-RulerBundle)
- [ ] [Contributions-Vim-Pp](https://github.com/hoaproject/Contributions-Vim-Pp)
- [ ] [Contributions-Zsh-Hoa](https://github.com/hoaproject/Contributions-Zsh-Hoa)
- [ ] [Core](https://github.com/hoaproject/Core)
- [ ] [Database](https://github.com/hoaproject/Database)
- [ ] [Devtools](https://github.com/hoaproject/Devtools)
- [ ] [Dispatcher](https://github.com/hoaproject/Dispatcher)
- [ ] [Dns](https://github.com/hoaproject/Dns)
- [ ] [Event](https://github.com/hoaproject/Event)
- [ ] [Eventsource](https://github.com/hoaproject/Eventsource)
- [ ] [Exception](https://github.com/hoaproject/Exception)
- [ ] [Fastcgi](https://github.com/hoaproject/Fastcgi)
- [ ] [File](https://github.com/hoaproject/File)
- [ ] [Graph](https://github.com/hoaproject/Graph)
- [ ] [Heap](https://github.com/hoaproject/Heap)
- [ ] [Http](https://github.com/hoaproject/Http)
- [ ] [Infrastructure](https://github.com/hoaproject/Infrastructure)
- [ ] [Irc](https://github.com/hoaproject/Irc)
- [ ] [Iterator](https://github.com/hoaproject/Iterator)
- [ ] [Json](https://github.com/hoaproject/Json)
- [ ] [Keynote](https://github.com/hoaproject/Keynote)
- [ ] [Kitab](https://github.com/hoaproject/Kitab)
- [ ] [Literature](https://github.com/hoaproject/Literature)
- [ ] [Locale](https://github.com/hoaproject/Locale)
- [ ] [Log](https://github.com/hoaproject/Log)
- [ ] [Mail](https://github.com/hoaproject/Mail)
- [ ] [Marvirc](https://github.com/hoaproject/Marvirc)
- [ ] [Math](https://github.com/hoaproject/Math)
- [ ] [Memory](https://github.com/hoaproject/Memory)
- [ ] [Mime](https://github.com/hoaproject/Mime)
- [ ] [Model](https://github.com/hoaproject/Model)
- [ ] [Notification](https://github.com/hoaproject/Notification)
- [ ] [Option](https://github.com/hoaproject/Option)
- [ ] [Paste](https://github.com/hoaproject/Paste)
- [ ] [Praspel](https://github.com/hoaproject/Praspel)
- [ ] [Promise](https://github.com/hoaproject/Promise)
- [ ] [Protocol](https://github.com/hoaproject/Protocol)
- [ ] [Prototype](https://github.com/hoaproject/Prototype)
- [ ] [Realdom](https://github.com/hoaproject/Realdom)
- [ ] [Regex](https://github.com/hoaproject/Regex)
- [ ] [Registry](https://github.com/hoaproject/Registry)
- [ ] [Router](https://github.com/hoaproject/Router)
- [ ] [Ruler](https://github.com/hoaproject/Ruler)
- [ ] [Serialize](https://github.com/hoaproject/Serialize)
- [ ] [Session](https://github.com/hoaproject/Session)
- [ ] [Shop](https://github.com/hoaproject/Shop)
- [ ] [Socket](https://github.com/hoaproject/Socket)
- [ ] [Static](https://github.com/hoaproject/Static)
- [ ] [Stream](https://github.com/hoaproject/Stream)
- [ ] [String](https://github.com/hoaproject/String)
- [ ] [Stringbuffer](https://github.com/hoaproject/Stringbuffer)
- [ ] [Test](https://github.com/hoaproject/Test)
- [ ] [Translate](https://github.com/hoaproject/Translate)
- [ ] [Tree](https://github.com/hoaproject/Tree)
- [ ] [Ustring](https://github.com/hoaproject/Ustring)
- [ ] [View](https://github.com/hoaproject/View)
- [ ] [Visitor](https://github.com/hoaproject/Visitor)
- [ ] [W3](https://github.com/hoaproject/W3)
- [ ] [Websocket](https://github.com/hoaproject/Websocket)
- [ ] [Worker](https://github.com/hoaproject/Worker)
- [ ] [Xml](https://github.com/hoaproject/Xml)
- [ ] [Xmlrpc](https://github.com/hoaproject/Xmlrpc)
- [ ] [Xyl](https://github.com/hoaproject/Xyl)
- [ ] [Zformat](https://github.com/hoaproject/Zformat)
- [ ] [Zombie](https://github.com/hoaproject/Zombie)

**Comment by Hywan** [comment]
Do you think it's a good idea to create a `Ci` repository, and use it as 
a submodule for all the other repositories? This `Ci` repository will 
contain scripts and/or files for CI, bots & co. The problem I'm trying 
to solve is avoid file duplication. Don't know if it would help.

**Comment by vonglasow** [comment]
Yes, it's a good idea if it was possible as Bors need to have a bors.toml file on root. I don't think it's possible for now.
